spec/sec-operations-on-objects-patch.html

```diff
@@ -0,0 +1,25 @@
+<emu-clause id="sec-operations-on-objects">
+  <h1>Operations on Objects</h1>
+  <ins>
+    <emu-clause id="sec-getoption" aoid="GetOption">
+      <h1>GetOption ( _options_, _property_, _type_, _values_, _fallback_ )</h1>
+      <emu-note>This is the same operation as <a href="https://tc39.es/ecma402/#sec-getoption">GetOption</a> in ECMA-402.</emu-note>
+      <p>
+        The abstract operation GetOption extracts the value of the property named _property_ from the provided _options_ object, converts it to the required _type_, checks whether it is one of a List of allowed _values_, and fills in a _fallback_ value if necessary. If _values_ is *undefined*, there is no fixed set of values and any is permitted.
+      </p>
+      <emu-alg>
+        1. Let _value_ be ? Get(_options_, _property_).
```

Comment:
```suggestion
        1. If Type(_options_) is Null or Undefined, then return _fallback_.
        1. Let _value_ be ? Get(_options_, _property_).
```

Reply:
`options` won't be `null` or `undefined`. It has to be a valid object before this operation is invoked.

Reply:
doesn't that mean that every caller has to conditionally call GetOption? That seems like a lot of repeated boilerplate.

Either way, either this change, or an assertion that it's Type Object, is needed.